includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
```

Comment:
Half want to change the signature of this, so we can receive an object... Would avoid some potential edge cases... Would be something like:
```php
function islandora_usage_stats_pid_to_db($pid) {
  $object = $pid instanceof AbstractObject ? $pid : islandora_object_load($pid);
  [...]
}
```

Then [below, where we have an object to pass,](https://github.com/Islandora/islandora_usage_stats/pull/50/files#diff-2d69368f59ba81d5635cd8274dd66090R21) we could do so and avoid the call to load again (and the possibility of this load failing, when we know we _had_ an object, at one point).